decoder.go

```diff
@@ -6,6 +6,10 @@ import (
 	"io"
 )
 
+var _ MNSDecoder = new(aliMNSDecoder)
+var _ MNSDecoder = new(aliMNSDecoderErrResp)
+var _ MNSDecoder = new(batchOpDecoder)
```

Comment:
Seems this `batchOpDecoder` doesn't have a corresponding `batchOpDecoderErrResp` so even the client is created with `aliMNSDecoderErrResp`, the returned error is not the errResp for some batch operations.

Reply:
Added support for batchOpDecoderErrResp in commit 2